esp-config/src/generate.rs

```diff
@@ -24,18 +25,27 @@ impl Value {
     fn parse_in_place(&mut self, s: &str) -> Result<(), ParseError> {
         *self = match self {
             Value::Bool(_) => match s {
-                "false" | "no" | "n" => Value::Bool(false),
-                "true" | "yes" | "y" => Value::Bool(true),
+                "false" | "no" | "n" | "f" | "FALSE" | "NO" | "N" | "F" => Value::Bool(false),
```

Comment:
We should convert s to lowercase before comparing, so we don't have to duplicate these and we can accept `FaLsE` as well.